scenariolib/visit.go

```diff
@@ -34,6 +34,7 @@ type Visit struct {
 	OriginLevel1       string
 	OriginLevel2       string
 	OriginLevel3       string
+	Referrer           string
```

Comment:
Add comments please. This is important because the technical doc is auto generated in Go.
@SLangevin